pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -53,6 +54,7 @@ type predicateMetadata struct {
 	serviceAffinityInUse               bool
 	serviceAffinityMatchingPodList     []*v1.Pod
 	serviceAffinityMatchingPodServices []*v1.Service
+	scalaResourcesHandleByExtenders    sets.String
```

Comment:
Please add a comment for this. Also mention that this is set based on the policy config and does not change per pod.